plugins/tracker-resources/src/components/NewIssueHeader.svelte

```diff
@@ -39,27 +40,52 @@
       closed = true
     })
   }
+  const dropdownItems: SelectPopupValueType[] = [
+    {
+      id: tracker.string.CreateProject,
+      label: tracker.string.CreateProject
+    },
+    {
+      id: tracker.string.NewIssue,
+      label: tracker.string.NewIssue
+    }
```

Comment:
We need resume draft if we in draft mode, and new issue here probable should not respect draft.